### ISSUE TITLE
supporting BIP 157

### ISSUE THREAD

**Issue by vv01f** [raspiblitz/raspiblitz#2465]
for enabling ([BIP 157](https://github.com/bitcoin/bips/blob/master/bip-0157.mediawiki)) connection from e.g. Breez, see https://github.com/breez/breezmobile/wiki/Connecting-a-Bitcoin-node-supporting-BIP-157

* [x] bitcoind to 0.21 (in raspiBlitzVersion 1.7.0)
* [ ] add "connect Breez" after [Tor support is available on Breez](https://github.com/breez/breezmobile/issues/322)
  + [ ] add to conf `blockfilterindex=1` and `peerblockfilters=1`

is there any conflict with current configuration details like `peerbloomfilters`?

**Comment by nyxnor** [comment]
I think Bisq requires peerbloomfilters=1

**Comment by openoms** [comment]
yes, Bisq needs peerbloomfilters, but there is no conflict with peerblockfilters.
you can edit your bitcoin.conf from the SYSTEM menu.
The connection will be over LAN only as Breez does not have Tor support bulit-in. You can experiment with Orbot on Android, but it is unlikely to be reliable.

**Comment by vv01f** [comment]
> yes, Bisq needs peerbloomfilters, but there is no conflict with peerblockfilters.

thx :) 

> The connection will be over LAN only as Breez does not have Tor support bulit-in.

yep, but it's on [roys roadmap](https://github.com/breez/breezmobile/issues/322), so we might add it to the connect wallets after that is working. :) and in direct contact he sounded motivated not to shelve the issue for too long.

**Comment by rookey** [comment]
Is this supposed to work already? I applied the changes to bitcoind config, but when trying to connect to my Raspi's IP, Breez still responds "Breez is unable to connect to the specified node. Please make sure this node supports BIP 157."
